packages/email/package.json

```diff
@@ -35,14 +35,14 @@
     "@react-email/section": "0.0.10",
     "@react-email/tailwind": "0.0.9",
     "@react-email/text": "0.0.6",
-    "nodemailer": "^6.9.3",
+    "nodemailer": "^6.9.9",
```

Comment:
This fixes the dependabot security alert here, https://github.com/documenso/documenso/security/dependabot/39

Reply:
Security alert didn't affect our usage but this is fine.